pkg/codegen/nodejs/gen.go

```diff
@@ -1197,7 +1204,7 @@ func (mod *modContext) genFunctionOutputVersion(
 
 	fmt.Fprintf(w, `
```

Comment:
The new line here (directly after `) is causing a blank line between the function comments and declaration.

Reply:
Fixed it ✅